packages/react-components/src/shared/src/augmentElement.tsx

```diff
@@ -0,0 +1,37 @@
+import { isString } from "lodash";
```

Comment:
Why rename to tsx? The code doesn't contains any JSX.

Reply:
```return <Type {...props} />;```

This is jsx!